app/helpers/configuration_helper/configuration_view_helper.rb

```diff
@@ -88,5 +88,13 @@ def list_view(resource)
        inactive_icon("view_tile.png", _('Tile View'), resource, "tile"),
        active_icon("view_list.png", _('List View'))].compact.join('')
     end
+
+    def check_roles(arr)
+      role = false
+      arr.each do |r|
+        role ||= role_allows(:feature => r)
+      end
+      role
```

Comment:
The code might also read better if the method name is something like `has_any_role` or something like that.